cli/cmd/plugin/diagnostics/Makefile

```diff
@@ -21,12 +21,11 @@ test: ## Run unit testing suite
 
 .PHONY: test-pr-check
 test-pr-check: ## Run e2e testing suite for PR check
-	./e2e-test/e2e-test.sh
+	cd ../../../.. && make diagnostic-e2e-test
 
 .PHONY: e2e-test
 e2e-test: ## Run e2e testing suite
-	./e2e-test/setup-e2e-test.sh
-	./e2e-test/e2e-test.sh
+	cd ../../../.. && make diagnostic-e2e-test
```

Comment:
This should call up to the root Makefile to call back down into the plugin makefile. Just have this perform the necessary calls to run the e2e tests and have the root makefile just call `make -C cli/cmd/plugin/diagnostics e2e-test`.